docs/rotate2.md

```diff
@@ -0,0 +1,44 @@
+## New Rotate
```

Comment:
As with the table of contents, I would propose "Rotate" instead of "New Rotate" just because it won't be new for long.